spec/plugin_mixin/aws_config_spec.rb

```diff
@@ -152,6 +152,25 @@ class DummyInputAwsConfigV2NoRegionMethod < LogStash::Inputs::Base
 
       end
     end
+
+  end
+
+  describe 'use aws bundled ca' do
+    context 'set to true' do
+      let(:settings) { { 'use_aws_bundled_ca' => true } }
+
+      it 'points ssl_ca_bundle to aws-sdk-core certs' do
+        expect(subject[:ssl_ca_bundle]).to match /aws-sdk-core.*ca-bundle\.crt/
```

Comment:
nit: end-of-string `\z` anchor

```suggestion
      it 'points ssl_ca_bundle to aws-sdk-core certs' do
        expect(subject[:ssl_ca_bundle]).to match /aws-sdk-core.*ca-bundle\.crt\z/
```